README.md

```diff
@@ -594,7 +604,7 @@ All statements above will return false if used with `===`
 
 <br/><br/>
 
-## ![✔] 5.6. Utilize all CPU cores
+## ![✔] 5.7. Utilize all CPU cores
```

Comment:
It's my preference to add new bullets to the bottom - otherwise bullet numbers in translation files no longer match up. Unsure if we agree on that though, so don't worry about changing it

Reply:
I wasn't sure either, that's why it in a separate commit :)
Do you want to wait for others to chime in?